### DISCUSSION TITLE
Coat/Bag check support

### DISCUSSION THREAD

**Discussion by reesericci** [pretix/pretix#3565] [Ideas]
I think it would be very helpful to be able to track customer's coats & bags for storage. I envision some kind of randm code being assigned to the customer & the bag that could be printed as a label that's affixed to the bag. I could then look up the customer and find the bag code to search in storage

**Comment by raphaelm** [comment]
I think this could very well be a separate, possibly integrated, system and is rather out of scope for pretix.
